output/pganalyze_collector/bloat_report.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.25.0
-// 	protoc        v3.12.3
+// 	protoc        v3.13.0
```

Comment:
I apparently have a newer `protoc`--should I roll this back and regenerate?

Reply:
I think its fine to update them